services/registry/src/sync/sync-gqlAuthProviders.ts

```diff
@@ -0,0 +1,14 @@
+import gqlObjects$ from "./sync-service";
+import { map, shareReplay, distinctUntilChanged, filter } from "rxjs/operators";
+
+
+const syncAuthProvider$ = gqlObjects$.pipe(
+    map(x => x.gqlauthproviders),
+    filter(a => a && Object.keys(a).length > 0),
+    distinctUntilChanged(),
```

Comment:
I think the distinct will work only if the `gqlauthproviders` are the same ref. If not, you should implement the equality function for the distinction